visualdl/logic/sdk.h

```diff
@@ -123,8 +123,11 @@ struct ScalarReader {
   ScalarReader(TabletReader&& reader) : reader_(reader) {}
 
   std::vector<T> records() const;
+  std::vector<T> records_addi(size_t start_index) const;
```

Comment:
Interface design  is not reasonable, need to re-design.
addi is not a good words to interpret the meaning